apps/server/src/modules/deletion/repo/deletion-request.repo.ts

```diff
@@ -31,11 +32,12 @@ export class DeletionRequestRepo {
 		await this.em.flush();
 	}
 
-	async findAllItemsToExecution(threshold: number, limit?: number): Promise<DeletionRequest[]> {
-		const currentDate = new Date();
-		const modificationThreshold = new Date(Date.now() - threshold);
-		const scope = new DeletionRequestScope().byDeleteAfter(currentDate).byStatus(modificationThreshold);
-		const order = { createdAt: SortOrder.desc };
+	async findAllItems(limit: number): Promise<DeletionRequest[]> {
```

Comment:
Would be good to add the `public` keyword and missing return types to prevent linter warnings.

Reply:
is so far a warning as far as I see
I would change it in all the files I touch, but this should be all at once.
I think is a refactoring, and we should do it all at once, quickly and for lot of files